wp-content/themes/borderzine/inc/widgets/class-borderzine-5-col-widget.php

```diff
@@ -0,0 +1,375 @@
+		</p>
+
+		<p>
+			<input class="checkbox" type="checkbox" <?php echo $showbyline; ?> id="<?php echo esc_attr( $this->get_field_id( 'show_byline' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'show_byline' ) ); ?>" /> <label for="<?php echo esc_attr( $this->get_field_id( 'show_byline' ) ); ?>"><?php esc_html_e( 'Show byline on posts?', 'largo' ); ?></label>
+		</p>
+
+		<p>
+			<input class="checkbox" type="checkbox" <?php echo $hidebylinedate; ?> id="<?php echo esc_attr( $this->get_field_id( 'hide_byline_date' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'hide_byline_date' ) ); ?>" /> <label for="<?php echo esc_attr( $this->get_field_id( 'hide_byline_date' ) ); ?>"><?php esc_html_e( 'Hide the publish date in the byline?', 'largo' ); ?></label>
+		</p>
+
+		<p>
+			<input class="checkbox" type="checkbox" <?php echo $show_top_term; ?> id="<?php echo esc_attr( $this->get_field_id( 'show_top_term' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'show_top_term' ) ); ?>" /> <label for="<?php echo esc_attr( $this->get_field_id( 'show_top_term' ) ); ?>"><?php esc_html_e( 'Show the top term on posts?', 'largo' ); ?></label>
+		</p>
+
+		<?php
+			// only show this admin if the "Post Types" taxonomy is enabled.
+			if ( taxonomy_exists( 'post-type' ) && of_get_option( 'post_types_enabled' ) ) {
+				?>
+					<p>
+						<input class="checkbox" type="checkbox" <?php echo $show_icon; ?> id="<?php echo esc_attr( $this->get_field_id( 'show_icon' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'show_icon' ) ); ?>" /> <label for="<?php echo esc_attr( $this->get_field_id( 'show_icon' ) ); ?>"><?php esc_html_e( 'Show the post type icon?', 'largo' ); ?></label>
+					</p>
+				<?php
+			}
+		?>
+
+		<p>
+			<strong><?php esc_html_e( 'Limit by Author, Categories or Tags', 'largo' ); ?></strong>
+			<br />
+			<small><?php esc_html_e( 'Select an author or category from the dropdown menus or enter post tags separated by commas (\'cat,dog\')', 'largo' ); ?></small>
+		</p>
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'author' ) ); ?>">
+				<?php esc_html_e( 'Limit to author: ', 'largo' ); ?>
+				<br />
+				<?php
+					wp_dropdown_users(
+						array(
+							'name' => $this->get_field_name( 'author' ),
+							'show_option_all' => __( 'None (all authors)', 'largo' ),
+							'selected' => $instance['author']
+						)
+					);
+				?>
+			</label>
+		</p>
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'prominence' ) ); ?>"><?php esc_html_e( 'Limit to prominence: ', 'largo' ); ?>
+				<?php
+					wp_dropdown_categories(
+						array(
+							'name' => $this->get_field_name( 'prominence' ),
+							'show_option_all' => __( 'None (all prominences)', 'largo' ),
+							'hide_empty' => 0,
+							'hierarchical' => 0,
+							'taxonomy' => 'prominence',
+							'selected' => $instance['prominence'],
```

Comment:
on a new instance of a widget:

> Notice: Undefined index: prominence in /wp-content/themes/borderzine/inc/widgets/class-borderzine-5-col-widget.php on line 311

so something like `'selected' => isset( $instance['prominence'] ) ? $instance['prominence'] : null,`

Reply:
Huh. Good catch. Updated in https://github.com/INN/umbrella-borderzine/pull/77/commits/46b3ddd9158ad9326ee215d2b3180a4ffb20b6a7

Reply:
This is apparently also happening in the 3 column recent posts widget merged here: https://github.com/INN/umbrella-borderzine/pull/67. I'll updated it in this PR

Reply:
Updated the 3 col widget in https://github.com/INN/umbrella-borderzine/pull/77/commits/101824c3f43964eaf53101554b3669f36a168da6

Reply:
both those, and the 6-column widget, are good now. 👍